### PR TITLE
Add unshallow flag to fetch

### PR DESCRIPTION
**Pull request by yulianedyalkova**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by MarcelMue — APPROVED** [review]
I missed this in the previous PR but let's add some changelog entries

File: pkg/git/git.go [hunk 1]
```diff
@@ -29,6 +29,7 @@ func Fetch(dir string) error {
 	// Fetch master so we can diff against it
 	argsArr := []string{
 		"fetch",
+		"--unshallow",
```
Comment (MarcelMue) [inline pkg/git/git.go:32]: Wait why is this actually needed?

Comment (yulianedyalkova) [reply pkg/git/git.go:32]: Because prow gives us a shallow copy of the releases repo (depth 1) and merge-base needs to have some history in order to work.

Comment (MarcelMue) [reply pkg/git/git.go:32]: Ahh okay - makes sense :+1:

**Pull request merged**